### PR TITLE
Add mobile ui operation breakdown

### PR DESCRIPTION
**Pull request by marandaneto**
Adding one more operation (mobile), related to https://github.com/getsentry/sentry/pull/24025

### CODE REVIEW

File: tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap [hunk 1]
```diff
@@ -10,6 +10,7 @@ breakdownsV2:
     - db
     - browser
     - resource
+    - ui
```
Comment (marandaneto) [inline tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap:13]: if this is auto-generated, it should probably be reverted? same for `test_project_config_with_breakdown/without_metrics.pysnap`

Comment (dashed) [reply tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap:13]: @marandaneto I'm unsure what you meant by that. Do you not want `ui` span op breakdowns be part of metrics?

**Review by dashed — APPROVED** [review]

**Comment by dashed** [comment]
@marandaneto Are you only interested in being able to query for these in Discover?

**Review by AbhiPrasad — APPROVED** [review]
See: https://develop.sentry.dev/sdk/performance/span-operations/

**Review by k-fish — APPROVED** [review]

File: tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap (hunk 1 above)
Comment (marandaneto) [reply tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap:13]: I meant that I just added a new `ui` value where I found the other operations in the code, I don't know the codebase so I'm not sure if `ui` should be added to those snapshot tests.

Comment (marandaneto) [reply tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap:13]: the same applies for the other snapshop test (test_project_config_with_breakdown/without_metrics.pysnap)
should I revert those 2 files or not?

Comment (dashed) [reply tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap:13]: I think it's fine.

@untitaker what do you think?

**Comment by dashed** [comment]
@marandaneto Is this still something you want to have? If so, what was preventing this from being merged after it was promptly approved?

**Review by untitaker — APPROVED** [review]
the snapshot changes are fine as-is

File: tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap (hunk 1 above)
Comment (marandaneto) [reply tests/sentry/relay/snapshots/test_config/test_project_config_with_breakdown/with_metrics.pysnap:13]: https://github.com/getsentry/sentry/pull/30363#pullrequestreview-842747571

**Comment by marandaneto** [comment]
> @marandaneto Is this still something you want to have? If so, what was preventing this from being merged after it was promptly approved?

sorry relay, I was OOO.
CI wasn't happy, that was the only blocker, let's try again.

**Comment by marandaneto** [comment]
@dashed @k-fish since this PR was never merged and CI was never totally happy due to other reasons not related to this PR, can we just merge it? I don't contribute to this repo so I'm unsure if this is ok or not.

**Comment by dashed** [comment]
@untitaker would you be able to assist on the metrics tests that are failing in this PR? 🙏

**Comment by untitaker** [comment]
fixed. but tbh this can be done by anybody with a functioning devsetup :)

**Comment by dashed** [comment]
@untitaker 🌮 

@marandaneto I think you can just rebase to `master`, and you should be able to _finally_ merge this PR. :D

**Comment by marandaneto** [comment]
> @untitaker 🌮
> 
> @marandaneto I think you can just rebase to `master`, and you should be able to _finally_ merge this PR. :D

relay test still fails :(

**Pull request merged**